Doc/update readme (#778)
CLIMADA-project/climada_python: README.md

CLIMADA stands for **CLIM**ate **ADA**ptation and is a probabilistic natural catastrophe impact model, that also calculates averted damage (benefit) thanks to adaptation measures of any kind (from grey to green infrastructure, behavioural, etc.).

> As of today, CLIMADA provides global coverage of major climate-related extreme-weather hazards at high resolution via a [data API](https://climada.ethz.ch/data-api/v1/docs) such as tropical cyclones, river flood, European winter storms, earthquakes, ..., all at 4km spatial resolution - wildfire to be added soon. For all hazards, historic and probabilistic event sets exist, for some also under select climate forcing scenarios (RCPs) at distinct time horizons (e.g. 2040). See also [papers](https://github.com/CLIMADA-project/climada_papers) for details.

Reviewer: ```suggestion
As of today, CLIMADA provides global coverage of major climate-related extreme-weather hazards at high resolution via a [data API](https://climada.ethz.ch/data-api/v1/docs) such as tropical cyclones, river flood, European winter storms, earthquakes, ..., all at 4km spatial resolution - wildfire to be added soon. For all hazards, historic and probabilistic event sets exist, for some also under select climate forcing scenarios (RCPs) at distinct time horizons (e.g. 2040).
You will find a repository containing scientific peer-reviewed articles that explain software components implemented in CLIMADA here: https://github.com/CLIMADA-project/climada_papers
```

It is recommend for new users to begin with the core (1) and the [tutorials](https://github.com/CLIMADA-project/climada_python/tree/main/doc/tutorial) therein.

> This is the Python (3.8+) version of CLIMADA - please see [https://github.com/davidnbresch/climada](https://github.com/davidnbresch/climada) for backward compatibility (MATLAB).

Reviewer: The link is already rendered as link? 🤔  Also, we're on Python 3.9 now
```suggestion
This is the Python (3.9+) version of CLIMADA - please see https://github.com/davidnbresch/climada for backward compatibility (MATLAB).
```

## Getting started

CLIMADA runs on Windows, macOS and Linux.
The released versions of the CLIMADA core can be installed directly through Anaconda:

```shell
conda install -c conda-forge climada
```

It is **highly recommended** to install CLIMADA into a **separate** Anaconda environment.
See the [installation guide](https://climada-python.readthedocs.io/en/latest/guide/install.html) for further information.

Follow the [tutorial](https://climada-python.readthedocs.io/en/stable/tutorial/1_main_climada.html) `climada_python-x.y.z/doc/tutorial/1_main_climada.ipynb` in a Jupyter Notebook to see what can be done with CLIMADA and how.

## Documentation

> Documentation is available on Read the Docs:

Reviewer: ```suggestion
The online documentation is available on Read the Docs: https://climada-python.readthedocs.io/en/stable/

The documentation of each release version of CLIMADA can be accessed separately through the drop-down menu at the bottom of the left sidebar. Additionally, the version 'stable' refers to the most recent release (installed via `conda`), and 'latest' refers to the latest unstable development version (the `develop` branch).
```

> Note that all the documentations has two versions,'latest' and 'stable', and explicit version numbers, 'vx.x.x', in the url path. 'stable' is created from the latest release (installed via conda) and 'latest' is created from the 'develop' branch and has the latest changes by developers. For more details about documentation versions, please have a look at [here](https://readthedocs.org/projects/climada-python/versions/).

Reviewer: ```suggestion
```

)

- *Cost-benefit analysis*: Bresch, D. N. and Aznar-Siguan, G., 2021: CLIMADA v1.4.1: towards a globally consistent adaptation options appraisal tool, Geosci. Model Dev., 14, 351-363, [https://doi.org/10.5194/gmd-14-351-2021](https://doi.org/10.5194/gmd-14-351-2021
)

and/or the following published articles:
- *Uncertainty and sensitivity analysis (unsequa)*:  Kropf, C. M. et al., 2022: Uncertainty and sensitivity analysis for probabilistic weather and climate-risk modelling: an implementation in CLIMADA v.3.1.0. Geoscientific Model Development 15, 7177–7201, [https://doi.org/10.5194/gmd-15-7177-2022](https://doi.org/10.5194/gmd-15-7177-2022
)

Aznar-Siguan, G. and Bresch, D. N., 2019: CLIMADA v1: a global weather and climate risk assessment platform, Geosci. Model Dev., 12, 3085–3097, https://doi.org/10.5194/gmd-12-3085-2019
- *LitPop exposures* : Eberenz, S., et al., D. N. (2020): Asset exposure data for global physical risk assessment. Earth System Science Data 12, 817–833, [https://doi.org/10.3929/ethz-b-000409595](https://doi.org/10.3929/ethz-b-000409595)

> Bresch, D. N. and Aznar-Siguan, G., 2021: CLIMADA v1.4.1: towards a globally consistent adaptation options appraisal tool, Geosci. Model Dev., 14, 351-363, https://doi.org/10.5194/gmd-14-351-2021

Reviewer: I suggest moving this into the documentation, maybe on its own page. I think this is too detailed information for a README. There, we can also provide, e.g., appropriate Bibtex entries
